Text/TextAnalyticsForHealth/azuredeploy.json

```diff
@@ -0,0 +1,143 @@
+      ],
+      "kind": "functionapp",
+      "properties": {
+        "name": "[variables('functionAppName')]",
+        "kind": "functionapp",
+        "httpsOnly": true,
+        "siteConfig": {
+          "appSettings": [
+            {
+              "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2021-04-01').keys[0].value)]"
+            },
+            {
+              "name": "WEBSITE_CONTENTSHARE",
+              "value": "[toLower(variables('functionAppName'))]"
+            },
+            {
+              "name": "AzureWebJobsDashboard",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2021-04-01').keys[0].value)]"
+            },
+            {
+              "name": "AzureWebJobsStorage",
+              "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccount'), ';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccount')), '2021-04-01').keys[0].value)]"
+            },
+            {
+              "name": "FUNCTIONS_EXTENSION_VERSION",
+              "value": "~3"
+            },
+            {
+              "name": "FUNCTIONS_EXTENSION_RUNTIME",
+              "value": "dotnet"
+            },
+            {
+              "name": "SCM_DO_BUILD_DURING_DEPLOYMENT",
+              "value": true
+            },
+            {
+              "name": "PROJECT",
+              "value": "[variables('functionProject')]"
+            },
+            {
+              "name": "TEXT_ANALYTICS_API_KEY",
+              "value": "[listKeys(resourceId('Microsoft.CognitiveServices/accounts', variables('textAnalyticsName')), '2021-04-30').key1]"
+            },
+            {
+              "name": "TEXT_ANALYTICS_API_ENDPOINT",
+              "value": "[concat('https://', variables('textAnalyticsName'), '.cognitiveservices.azure.com')]"
+            }
+          ]
+        }
+      },
+      "resources": [
+        {
+          "type": "sourcecontrols",
+          "apiVersion": "2015-08-01",
+          "name": "web",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites', variables('functionAppName'))]"
+          ],
+          "properties": {
+            "RepoUrl": "[variables('repoURL')]",
+            "branch": "[variables('repoBranch')]",
+            "project": "[variables('functionProject')]",
+            "IsManualIntegration": true
+          }
+        }
+      ]
+    }
+  ],
+  "outputs": {
+    "Azure Function Site Name": {
+      "type": "String",
+      "value": "[variables('functionAppName')]"
+    }
+  }
```

Comment:
I would love to see if there is a way to output the function auth code in ARM templates now (it wasn't possible before which is why other power skills don't do it). Not a big deal if you want to just file a follow up work item for it though

Reply:
I was able to get it to output the full URI for the skill. should be resolved but for some reason this block doesn't show as `outdated`. Let me know if you can't find that change and I will make sure it gets there. :)

Reply:
Wooo so awesome! This will be sooooo easy for customers to use now :D We need to update the other power skills with all of these improvements @bleroy